packages/ubuntu_desktop_installer/lib/routes.dart

```diff
@@ -7,4 +7,5 @@ abstract class Routes {
   static const allocateDiskSpace = '/allocatediskspace';
   static const updatesOtherSoftware = '/updateothersoftware';
   static const writeChangesToDisk = '/writechangestodisk';
+  static const chooseYourLook = 'chooseyourlook';
```

Comment:
For consistency, that route should start with a forward slash.

Reply:
@oSoMoN fixed, sorry :)